libraries/ms-common/inc/log.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+#include <stdio.h>
+
+typedef enum {
+  LOG_LEVEL_DEBUG = 0,
+  LOG_LEVEL_WARN,
+  LOG_LEVEL_FAULT,
```

Comment:
Nit: the typical levels I see tend to see in other libraries are

- `DEBUG`
- `INFO`
- `WARNING` or `WARN`
- `ERROR` or `ERR`
- `FATAL` or `CRITICAL`

We definitely don't need all of these but personally, I'd rather see `ERROR` or `FATAL` or `CRITICAL` than `FAULT`. This is personal preference but it seems pretty standard common Golang, Python, Java and C++ logging libraries all use those and we might as well follow the convention where we can.